ui/app/pages/settings/settings.component.js

```diff
@@ -136,7 +136,9 @@ class SettingsPage extends PureComponent {
           className={c({ 'settings-page__subheader--link': initialBreadCrumbRoute })}
           onClick={() => initialBreadCrumbRoute && history.push(initialBreadCrumbRoute)}
         >{subheaderText}</div>
-        {breadCrumbTextKey && <div><span>{'> '}</span>{t(breadCrumbTextKey)}</div>}
+        {breadCrumbTextKey && <div
+          className={c({ 'settings-page__subheader--break': initialBreadCrumbRoute })}
+        ><span>{' > '}</span>{t(breadCrumbTextKey)}</div>}
         {isAddressEntryPage && <div><span>{' > '}</span>{addressName}</div>}
```

Comment:
The same class might be useful to apply here as well. The address entry page appears to suffer from the same problem

Reply:
updated based on feedback